### PR TITLE
Update client.js

### PR DESCRIPTION
**Pull request by Mr-Catty**
By Mr. Cat

## Type
- [x] Functions: $client
- [ ] Callbacks: None
- [ ] Handlers: None
- [ ] Others: None
- [ ] Bug Fix: None

Git commit: [Here](https://github.com/Mr-Catty/Aoi.js-by-cat.git)

Want a credit? Discord tag or other social media links: Pranav#1758

## Description
Added some properties for $client function.

### CODE REVIEW

File: package/functions/funcs/client.js [hunk 1]
```diff
@@ -73,6 +85,36 @@ module.exports = async (d) => {
...
       option = d.client.presence.status;
       break;
+    case "token":
+        option = d.client.token;
+        break;
+    case "verified":
+        option = d.client.user.verified;
+        break;
+    case "clienttag":
+        option = d.client.user.username + '#' + d.client.user.discriminator;
```
Comment (XzFirzal) [inline package/functions/funcs/client.js:95]: Alias of "tag"

File: package/functions/funcs/client.js [hunk 2]
```diff
@@ -73,6 +85,36 @@ module.exports = async (d) => {
...
+        break;
+    case "rawavatar":
+        option = d.client.user.avatar;
+        break;
+    case "mfaenabled":
```
Comment (XzFirzal) [inline package/functions/funcs/client.js:100]: I see no point of this

File: package/functions/funcs/client.js [hunk 3]
```diff
@@ -73,6 +85,36 @@ module.exports = async (d) => {
...
+        option = d.client.user.mfaEnabled;
+        break;
+    case "dmchannel":
```
Comment (XzFirzal) [inline package/functions/funcs/client.js:103]: I see no point of this

File: package/functions/funcs/client.js [hunk 4]
```diff
@@ -73,6 +85,36 @@ module.exports = async (d) => {
...
+        option = d.client.user.dmChannel;
+        break;
+    case "ownerid":
+        option = d.client.ownerID;
+        break;
+    case "teamid":
+        option = bapp.owner;
+        break;
+    case "partial":
```
Comment (XzFirzal) [inline package/functions/funcs/client.js:112]: I see no point of this

File: package/functions/funcs/client.js (hunk 2 above)
Comment (Mr-Catty) [reply package/functions/funcs/client.js:100]: It's multi factor authentication.

**Pull request merged**